packages/venia-ui/lib/components/WishlistPage/wishlistPage.js

```diff
@@ -24,7 +24,11 @@ const WishlistPage = props => {
     const wishlistElements = useMemo(() => {
         if (wishlists.length === 0) return null;
         return wishlists.map(wishlist => (
-            <Wishlist key={wishlist.id} data={wishlist} />
+            <Wishlist
+                key={wishlist.id}
+                data={wishlist}
+                shouldRenderVisibilityToggle={wishlists.length > 1}
```

Comment:
Would it merit if we create and send it over from the talon instead? By doing so we are opening up the component to have custom definitions for `shouldRenderVisibilityToggle`. In the default case, it will be `wishlists.length > 1`. What do you think?

Reply:
Sure, sounds good.